docs/mobile/state-management.md

```diff
@@ -1,7 +1,85 @@
 ## Mobile Playbook: State management
 
-#### Persist Redux/Flux store to disk to survive shutdowns and tombstoning
+State management strategies in React Native are similar to regular React applications: Flux architecture with [Redux](http://redux.js.org/) is a sensible default choice. Alternatives like [MobX](https://github.com/mobxjs/mobx) and [Relay](https://facebook.github.io/relay/) are also interesting, but not discussed at length here.
+
+#### Persist Redux/Flux store to disk
+
+Most mobile applications will need to persist their data stores to achieve a snappy user experience and offline resilience.
+
+The simplest way to achieve persistent state and offline caching is to serialize the store object to JSON and write it to disk via the [AsyncStorage](https://facebook.github.io/react-native/docs/asyncstorage.html) API, and reset the store state from the serialized representation on app startup.
+
+You can use the [redux-storage](https://github.com/michaelcontento/redux-storage) middleware with the [redux-storage-engine-reactNativeAsyncStorage](https://github.com/michaelcontento/redux-storage-engine-reactNativeAsyncStorage) backend to achieve this. For performance reasons, [debouncing](https://github.com/michaelcontento/redux-storage-decorator-debounce) the serialization is usually a good idea.
 
 #### Prepare for store schema migrations/pruning on updates
 
+When you persist your application state to disk, you have created a database. And like traditional databases, the schema and existing data may need to be migrated when new versions of application code are released.
+
+Before launching your application to the public, consider what should happen to the persisted application state when the newly updated app is first launched.
+
+##### Pruning
+
+In some cases the application state is a cache that can be repopulated from the backend. Easiest solution is to jettison the persisted state and allow the new version of the application to start from empty initial state.
+
+Sometimes the state can be mostly discarded, but some bits like user session information needs to be kept to avoid having the user to log into the application again. Simple solution is to organise the redux store in a way that some subtrees can be kept as-is, and others discarded wholesale.
```

Comment:
>...some bits like user session information needs to be kept to avoid having the user to log into the application again

Phrasing is a little awkward here. How about:

>some bits like user session information needs to be kept, so the user isn't required to log into the application again.